analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -145,8 +145,7 @@ class GoMod(
      * Return the module graph output from `go mod graph` with non-vendor dependencies removed.
      */
     private fun getModuleGraph(projectDir: File): Graph {
-        val moduleInfoForModuleName = getModuleInfos(projectDir).map { it.replace ?: it }
-            .associateBy({ it.path }, { it })
+        val moduleInfoForModuleName = getModuleInfos(projectDir).associateBy({ it.path }, { it })
```

Comment:
Commit message:
- Maybe "for the" -> "of the" (in the title)
- "all use cases want to view" sounds a bit strange IMO, as if use cases were persons. Maybe "all callers assume the 'replace' directive to have been applied" or so?